client/components/section-nav/style.scss

```diff
@@ -16,6 +16,10 @@
 		visibility: hidden;
 	}
 
+	&.is-hidden {
+		border: none !important;
```

Comment:
How about using `visibility: hidden;` as above? Or could we change the selector to match the following one? Using `!important` here seems too powerful 🤔

```
.main.hosting-dashboard-layout.sites-dashboard.sites-dashboard__layout .hosting-dashboard-item-view .hosting-dashboard-item-view__navigation
```

Reply:
I tried `visibility: hidden;` but somehow it still makes the element to have nonzero height due to the border. Let me think...

Reply:
Fixed via https://github.com/Automattic/wp-calypso/pull/99998/commits/fffb093d5082416db5bd822a31ce6f1ca921a4ce!